pygnmi/client.py

```diff
@@ -471,12 +471,19 @@ def get(
             raise gNMIException("Conversion of gNMI paths to the Protobuf format failed", e)
 
         try:
-            gnmi_message_request = GetRequest(
-                prefix=protobuf_prefix,
-                path=protobuf_paths,
-                type=pb_datatype,
-                encoding=pb_encoding,
-            )
+            if prefix is None and target is None:
+                gnmi_message_request = GetRequest(
+                    path=protobuf_paths,
+                    type=pb_datatype,
+                    encoding=pb_encoding,
+                )
+            else:
+                gnmi_message_request = GetRequest(
+                    prefix=protobuf_prefix,
+                    path=protobuf_paths,
+                    type=pb_datatype,
+                    encoding=pb_encoding,
+                )
```

Comment:
the protobuf being empty should be handled by target. What is the error here we are getting ?
It will be great if you help us to point in gNMI spec where it indicates this behavior.